### ISSUE TITLE
Not working properly, Luma3DS crashes after RR 3ds loading logo.

### ISSUE THREAD

**Issue by NoahARKADE** [SleepyLark/RidgeRadio#1]
Tried to use it with ModMoon, doing so it detected the mod but when i went to load the game my 3ds had a fatal crash and no longer can boot the game past the loading screen even on the 3ds main menu.

**Comment by SleepyLark** [comment]
I've been getting a similar error but only when trying to start a race. If I had to guess something broke after an update with Luma3DS that's affecting LayeredFS patching. Try updating to the latest version and see if anything changes. If not, the problem must be that the audio files are either too large or need to be reconverted ;-;

**Comment by SleepyLark** [comment]
Random question, but have you tried disabling the forced L2 cache and clock settings in Luma? I'm curious if it's the L2 cache that's causing issues as mentioned [here](https://www.reddit.com/r/3dshacks/comments/l89y9d/comment/gp2dq87/?utm_source=share&utm_medium=web2x&context=3).

**Comment by SleepyLark** [comment]
@NoahARKADE 
Okay so I did a little testing a may have found a way to fix it. Could you please update to the latest version of Luma3DS and turn off the force clock speed and L2 cache either in the main setting or in Rosalina and see if that fixes it. That seemed to fix the problem I had with it crashing on my New 3DS.

**Comment by vezix** [comment]
![WhatsApp Image 2023-12-18 at 15 24 50](https://github.com/SleepyLark/RidgeRadio/assets/36164109/2b54eb40-276f-47a3-8093-9daab999fb83)
 

encounter this instead

**Comment by SleepyLark** [comment]
@NoahARKADE 
@vezix 

Please update to the newest version of Luma3DS try turning off the New 3DS options (if applicable) and disabling the plugin loader in the Rosalina menu.

**Comment by simonkyleninja** [comment]
Hey are you the one that made this mod. I hope you don't mind me asking. Is it possible to put other custom music on this game. I'm a huge fan of ridge racer 64. I love the soundtrack but every time I put it in a long with the others already install. It doesn't show the name of the track. Any ideas, why this would be the case? I would appreciate it a lot.

**Comment by SleepyLark** [comment]
> Hey are you the one that made this mod. I hope you don't mind me asking. Is it possible to put other custom music on this game. I'm a huge fan of ridge racer 64. I love the soundtrack but every time I put it in a long with the others already install. It doesn't show the name of the track. Any ideas, why this would be the case? I would appreciate it a lot.

Yes but they need to be converted to .bcstm and filename renamed to match game files so it plays that instead (filenames and tracks are listed in the Google Sheet on the front page). 

As for showing the track names I tried seeing if it was possible but wasn't able to figure it out. I need to find my old laptop that I used to first make the mod since it has everything I need on there instead of starting from scratch again. Would be a nice feature though

**Comment by simonkyleninja** [comment]
Yeah I tried that. I tried to convert them into bcstm's using loup audio
converter and then rename them to match the game file so that it recognize
them but it won't play on the track. Are there specific requirements like
the hz rate and the duration of the track?

**Comment by SleepyLark** [comment]
> Yeah I tried that. I tried to convert them into bcstm's using loup audio
> converter and then rename them to match the game file so that it recognize
> them but it won't play on the track. Are there specific requirements like
> the hz rate and the duration of the track?
> […](#)

Which version of Looping Audio Converter are you on? I know some of the newer version weren't working but I was hoping latest one would have fixed it. If that's the still the case then try this version, I believe it's the one I used originally 

https://www.lakora.us/brawl/loopingaudioconverter/

If you have still have problems then open a new issue on Github

**Comment by SleepyLark** [comment]
Closing due to lack of response from the original user and not having issues on my end using the latest version of Luma
